models/Context.php

```diff
@@ -353,9 +355,12 @@ private function getParents($class)
     private function getInterfaces($class)
     {
         $interfaces = [];
-        foreach($class->interfaces as $interface) {
-            if (isset($this->interfaces[$interface])) {
-                $interfaces[] = $this->interfaces[$interface];
+
+        if (!empty($class->interfaces)) {
```

Comment:
Along the same lines of https://github.com/yiisoft/yii2-apidoc/pull/142/files#r106962261, `Context->inheritMethodRecursive()` gets around to calling `$this->getInterfaces($class)` where $class is of type `InterfaceDoc` (referring to our `ElementQueryInterface` and `$method` is referring to our `one()`.  But `InterfaceDoc` doesn't have a `interfaces` property to loop over.

Which again, makes me think that the proper solution for this and the previous comment might be to for `inheritMethodRecursive()` to not call `$this->getInterfaces()` on `InterfaceDocs` classes?


    $inheritanceCandidates = array_merge(
        $this->getParents($class),
        $this->getInterfaces($class)
    );

Reply:
Same as https://github.com/yiisoft/yii2-apidoc/pull/142/files#r315932500